src/pyasm/prod/service/api_xmlrpc.py

```diff
@@ -2711,7 +2711,16 @@ def get_paths(my, ticket, search_key, context="publish", version=-1, file_type='
 
     @xmlrpc_decorator
     def get_base_dirs(my, ticket):
-        '''get all of the base directories defined on the server'''
+        '''get all of the base directories defined on the server.'''
+        
+        from pyasm.security import Site
+        site = Site.get()
+        if site:
+            data = {}
+            data['asset_base_dir'] = site.get_asset_dir(alias="default")
+            data['web_base_dir'] = site.get_web_dir(alias="default")
+            return data
+        
```

Comment:
what about the other ones like mac_client_dir or sandbox_dir.. why does this only contain only 2 items?